src/views/partials/createdcards.ejs

```diff
@@ -0,0 +1,16 @@
+<% for(let i=0; i<created.length; i++){ %>
```

Comment:
It will be more natural to make a component for an item rather than a list of items. Because an item is easier to be re-used comparing to a list.

Moreover, the three types of card are so similar. Please refactor and combine them into one card component. Think about how to do it with a correct sense of abstraction layer. One definitely wrong example is to have logic that compare job status inside the card component.